code/modules/antagonists/_common/antag_datum.dm

```diff
@@ -59,6 +61,31 @@ GLOBAL_LIST_EMPTY(antagonists)
 /datum/antagonist/proc/remove_innate_effects(mob/living/mob_override)
 	return
 
+// Adds the specified antag hud to the player. Usually called in an antag datum file
+/datum/antagonist/proc/add_antag_hud(antag_hud_type, antag_hud_name, mob/living/mob_override)
+	var/datum/atom_hud/antag/hud = GLOB.huds[antag_hud_type]
+	hud.join_hud(mob_override)
+	set_antag_hud(mob_override, antag_hud_name)
+
+
+// Removes the specified antag hud from the player. Usually called in an antag datum file
+/datum/antagonist/proc/remove_antag_hud(antag_hud_type, mob/living/mob_override)
+	var/datum/atom_hud/antag/hud = GLOB.huds[antag_hud_type]
+	hud.join_hud(mob_override)
+	set_antag_hud(mob_override, null)
+
+// Handles adding and removing the clumsy mutation from clown antags. Gets called in apply/remove_innate_effects
+/datum/antagonist/proc/handle_clown_mutation(mob/living/mob_override, message)
+	var/mob/living/carbon/human/H = mob_override
+	if(H.mind.assigned_role == "Clown")
+		if(H && istype(H))
+			if(H.dna.check_mutation(CLOWNMUT)) // They're a clown becoming an antag, remove clumsy
```

Comment:
I'd prefer if it was explicit if we're adding or removing it, nothing stops clown that lost it before in some way getting antag later.

Reply:
Should I make two seperate procs to add and remove?

Reply:
That or pass in the new value, either way works.

Reply:
Which new value do you mean?

Reply:
```
handle_clown_mutation(x,y,remove=TRUE)
  if(add)
    add mutation
  else
    remove mutation
``` Or whatever configuration of parameters will be used most often.

Reply:
Alright, made some changes, hopefully it makes sense